cypress/integration/pages/testsForAllPages.js

```diff
@@ -53,11 +54,113 @@ const testsForAllPages = ({ service, pageType }) => {
-            appID: '1609039196070050',
+        it('should have the correct shared metadata', () => {
+          cy.get('head').within(() => {
+            cy.get('meta[name="fb:admins"]').should(
+              'have.attr',
+              'content',
+              '100004154058350',
+            );
+            cy.get('meta[name="fb:app_id"]').should(
+              'have.attr',
+              'content',
+              '1609039196070050',
+            );
+            cy.get('meta[name="og:image"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].defaultImage,
+            );
+            cy.get('meta[name="og:image:alt"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].defaultImageAltText,
+            );
+            cy.get('meta[name="og:locale"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].locale,
+            );
+            cy.get('meta[name="og:site_name"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].defaultImageAltText,
+            );
+            cy.get('meta[name="og:type"]').should(
+              'have.attr',
+              'content',
+              pageType === 'articles' ? 'article' : 'website',
+            );
+            cy.get('meta[name="og:url"]').should(
+              'have.attr',
+              'content',
+              `https://www.bbc.com${config[service].pageTypes[pageType].path}`,
+            );
+            cy.get('meta[name="og:site_name"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].brandName,
+            );
+            cy.get('meta[name="twitter:card"]').should(
+              'have.attr',
+              'content',
+              'summary_large_image',
+            );
+            cy.get('meta[name="twitter:creator"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].twitterCreator,
+            );
+            cy.get('meta[name="twitter:image:alt"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].defaultImageAltText,
+            );
+            cy.get('meta[name="twitter:image:src"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].defaultImage,
+            );
+            cy.get('meta[name="twitter:site"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].twitterSite,
+            );
           });
+          cy.request(`${config[service].pageTypes[pageType].path}.json`).then(
+            ({ body }) => {
+              const description =
+                pathOr(null, ['promo', 'summary'], body) ||
+                pathOr(null, ['promo', 'headlines', 'seoHeadline'], body) ||
+                pathOr(null, ['metadata', 'summary'], body);
+              const title =
```

Comment:
This is not null but "null" for naidheachdan. I have investigated this. It's not only a problem in our local fixture data, but a source data problem in the test (but not live environment) too. 

To check for yourself, go here: https://www.test.bbc.co.uk/naidheachdan inspect and search for og:title or twitter:title

Reply:
Same requests for issues as above apply, though this is no reproducible on live env.